apps/desktop/tsconfig.json

```
@@ -28,6 +28,7 @@
       "@bitwarden/vault-export-ui": ["../../libs/tools/export/vault-export/vault-export-ui/src"],
       "@bitwarden/importer/core": ["../../libs/importer/src"],
       "@bitwarden/importer/ui": ["../../libs/importer/src/components"],
+      "@bitwarden/km/biometrics": ["../../libs/key-management/src/biometrics"],
```

I think we should be more explicit on the team name.
Also why not just provide root path to `/src` ?:
```suggestion
      "@bitwarden/key-management": ["../../libs/key-management/src"],
```

---

Do we want to put all exports under just one path? Other seems seem to divide these. If it's about the amount of exports, we could ` "@bitwarden/key-management/*": ["./libs/key-management/src/*"],`.

I'll change the km to key-management.

---

As you can see from other exports, it is structured in a way, that each is separate module (package.json).
I prefer instead one of the two options:
- `"@bitwarden/key-management/*"`, then accessible with `import { BiometricsService } from "@bitwarden/key-management/biometrics"` (easier to do)
- `"@bitwarden/key-management"`, with accompanying `index.ts` within the module, which provides explicit list of exports, then accessible with `import { BiometricsService } from "@bitwarden/key-management"` (more correct way ?)

---

Fair, that makes sense. I'm still not sure about importing so many different domains (biometrics, general km, proc reload and so on) from a single package. But I suppose we can refactor if this actually causes issues.